docs/guides/setup/windows.md

```diff
@@ -21,8 +21,8 @@ but can be unexpected.
 6. Install nvm via their install and update script
    [found here](https://github.com/nvm-sh/nvm#install--update-script)
 7. Source your newly NVMed .bashrc `source ~/.bashrc`
-8. Install Node 10 `nvm install 10`
-9. Install Python 2 `sudo apt install python-minimal`
+8. Install Node 12 `nvm install 12`
```

Comment:
```suggestion
8. Install latest LTS Node release `nvm install --lts`
```

does this help us avoid bumping the version number in the docs every time it updates?

Reply:
Good idea!